src/plugins/intel_gpu/src/graph/impls/ocl/gather_nd.cpp

```diff
@@ -23,6 +23,7 @@ struct gather_nd_impl : typed_primitive_impl_ocl<gather_nd> {
         return make_unique<gather_nd_impl>(*this);
     }
 
+public:
```

Comment:
Why did you add this? It is already "public".

Reply:
FIxed. Thanks!